src/components/Auth/Auth.tsx

```diff
@@ -104,11 +134,52 @@ function Auth() {
           <ModeSwitch className="signupLoginText">
             <Typography.Text>{signup ? 'Already have an account?' : "Don't have an account?"}</Typography.Text> <br />
             <Typography.Link onClick={handleModeSwitch}>
-              {signup ? 'Log in' : 'Limited Time First 50 $MAGIC✨ Hour Pod Listeners get 200 FREE credits'}
-            </Typography.Link>
+              {signup ? 'Log in' : 'Limited time First 87 $MAGIC✨ users Free sign up'}
+            </Typography.Link>{' '}
+            <br /> <br />
+            {!signup && <Typography.Link onClick={handlePasswordReset}>Forgot Password?</Typography.Link>}
           </ModeSwitch>
         )}
       </Form>
+      <Modal
+        title="Reset Password"
+        centered
+        className="modal"
+        visible={showPwResetModal}
+        onCancel={handleCancel}
+        footer={[
+          <Button
+            key="submitResetPassword"
+            type="primary"
+            disabled={!pwResetLogin}
```

Comment:
what I meant was to use the `resetting` state variable to disable this when you click submit, just like any other form